test/helper.rb

```diff
@@ -21,4 +21,4 @@
 require 'minitest/pride'
 
 require 'webmock/test_unit'
-WebMock.disable_net_connect!
+WebMock.enable_net_connect!
```

Comment:
You shouldn't do this.
Please revert this line.

Reply:
Just reverted. Sorry, I'm kinda new with these tools. The tests were doing fine with this directive locally, I didn't know I shouldn't change that line. 
I updated the PR message, as the work in its current state is clearly not passing the tests. I will look for errors and shortcomings in my code in the upcoming days.